website/content/labs/lab7.md

```diff
@@ -180,33 +167,31 @@ Before continuing, make sure that you also push to your GitHub repository:
 
 
-# Task 3: Create .travis.yml for cs220-helloapp-2019
+# Task 3: Create .travis.yml for HelloApp
 [40 points]
 
-As you remember from our [previous lab]({{< relref "lab6.md" >}}), we have a way to make sure our app passes all the tests every time we push.
+As you may remember from our [previous lab]({{< relref "lab6.md" >}}), we have a way to make sure our app passes all the tests every time we push.
 
-For this, you should create a `.travis.yml` file in your `cs220-helloapp-2019` repo. With python, there's no need to build, so you should just define a test phase which runs the command `pytest` as its test script.
+For this, you should create a `.travis.yml` file in your `2020-lab7-GITHUB_USERNAME` repo. With Python, there's no need to build, so you should just define a single job that runs the command `pytest` as its test script.
 
-In the last task, you made the tests fail. You should ensure the CI job fails when the tests are failing.
+In the last task, you made the tests fail. You should ensure the CI job fails when the tests are failing. Take the URL of the failed build, and enter it in Gradescope.
 
-Now, go back and fix the app so that they all pass. Before continuing, make sure you push your fixed app to both GitHub and to Heroku.
+Now, go back and fix the app so that they all pass. Then, push your fixed app to both GitHub and to Heroku. Take the URL of the successful build, and enter it in Gradescope.
 
 # Task 4: Deploy using Github Integration with Travis CI
 [20 points]
 
-Wouldn't it be convenient if we could deploy continuously, as soon as tests pass? You might think "well, what if we had a deploy phase in our Travis CI"? This is possible (see https://docs.travis-ci.com/user/deployment/heroku/), but it turns out Heroku makes it *even easier* than that. On your Heroku app dashboard, in the "Deploy" tab, select "GitHub - connect to GitHub" instead of "Heroku Git - Use Heroku CLI".
+Wouldn't it be convenient if we could deploy continuously, as soon as tests pass? You might think "well, what if we had a deploy phase in our Travis CI"? This is possible (see https://docs.travis-ci.com/user/deployment/heroku/), but it turns out Heroku makes it *even easier* than that. On your Heroku app dashboard, in the "Deploy" tab, under "Deployment method", select "GitHub - connect to GitHub" instead of "Heroku Git - Use Heroku CLI".
 
-Then, you should be able to connect to GitHub using your GitHub account and repo name. Once you connect the repo, you should see an option that says "Automatic deploys" with a checkbox "Wait for CI to pass before deploy".
+Then, you should be able to connect to GitHub using your GitHub account and repo name. Once you connect the repo, you should see an option that says "Automatic deploys" with a checkbox "Wait for CI to pass before deploy". Make sure that checkbox is checked, and then click on "Enable Automatic Deploys"
```

Comment:
Finding the correct repo was a bit tricky for me. I had to click on the dropdown menu, pick the `cmsc22000-labs` organization, and then search `2020-lab7-CNETID` to find my repo. [Here's a screenshot](https://i.imgur.com/BocACnr.png). I think we should provide more guidance for this step.

Reply:
I've added more concrete guidance here.